src/components/AssetInfo/TokenSideBarBeneficiary.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import css from "./TokenSideBar.scss";
+import TokenSideBarField from "./TokenSideBarField";
+
+const TokenSideBarBeneficiary = () => (
+  <TokenSideBarField title="Allow Transfer" ctaText="Allow" ctaStatus="success">
+    <label>
```

Comment:
Make the title to "Approve Change Beneficiary". and submit button text to "Approve".